src/collective/behavior/banner/browser/viewlets.py

```diff
@@ -96,3 +102,44 @@ def random_banner(self):
 
         random.shuffle(banners)
         return banners
+
+    def getVideoEmbedMarkup(self, obj):
```

Comment:
@fulv You don't need the obj here (also obj is no object nut only a dict here), better only pass the url.